test/dlc_tests/container_tests/bin/test_efa_sanity.py

```diff
@@ -0,0 +1,62 @@
+import subprocess
```

Comment:
If this is a sanity test, then it should be invoked in `test/dlc_tests/sanity`.

Reply:
Makes sense. invoking it from sanity.